### PR TITLE
New app drawer card for ld track

### PR DESCRIPTION
**Pull request by creilly8**
## Description
I wasn't sure what to put for the description. Feel free to make suggestions. 

## Checklist

[Check each task](https://github.com/stjude/proteinpaint/wiki/Pull-Request-Checklist) that has been performed or verified to be not applicable.
- [x] Tests: added and/or passed unit and integration tests, or N/A
- [x] Todos: commented or documented, or N/A
- [x] Notable Changes: updated release.txt, prefixed a commit message with "fix:" or "feat:", added to an internal tracking document, or N/A

### CODE REVIEW

**Comment by gavrielm** [comment]
I think you need to also show a track for variants in the app drawer card because the LD values were computed between variants.

Possible description could be: "The LD track shows linkage disequilibrium (LD) values between variant pairs. LD values represent r<sup>2</sup> coefficients. LD computations were performed separately for each ancestry".

@josephpowi generated the LD data, so he might also have suggestions for the description.

File: front/public/cards/ld.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+                "block": true,
+                "nobox": 1,
+                "noheader": 1,
+                "genome": "hg38",
+                "nativetracks": "Refgene",
+                "tracks": [
+                    {
+                        "type": "ld",
+                        "name": "European ancestry",
+                        "file": "files/hg38/sjlife/ld/CEU.gz",
```
Comment (xzhou82) [inline front/public/cards/ld.json:17]: please replace this actual dataset file with a slice file and load it from tp/proteinpaint_demo/hg38/LD/

File: front/public/cards/ld.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
+{
```
Comment (xzhou82) [inline front/public/cards/ld.json:1]: change file name to LD.json

**Comment by josephpowi** [comment]
> LD values represent r2 coefficients. LD computations were performed separately for each ancestry

A small change. Please have a look, and we can discuss more if needed. 
The LD (linkage disequilibrium) track shows the non-random association of alleles at different loci in a given population.

**Review by xzhou82 — APPROVED** [review]

**Pull request merged**